core/src/trezor/ui/components/common/confirm.py

```diff
@@ -1,16 +1,26 @@
-from trezor import loop, ui
+from trezor import loop, ui, wire
 
 if __debug__:
     from apps.debug import confirm_signal
 
 if False:
-    from typing import List, Tuple, Optional
+    from typing import List, Tuple, Optional, Any, Awaitable
 
 CONFIRMED = object()
 CANCELLED = object()
 INFO = object()
 
 
+def is_confirmed(x: Any) -> bool:
+    return x is CONFIRMED
+
+
+async def raise_if_cancelled(a: Awaitable, exc: Any = wire.ActionCancelled) -> None:
```

Comment:
the type should probably be `BaseException | Type[BaseException]` (here and in the other places that pass this through

Reply:
`wire.ActionCancelled` is an `int` though?

Reply:
i'm somewhat sure it's an `Error`

Reply:
D'oh, was looking `message/FailureType.py`. Maybe I should try something more intelligent than `ctags`.:thinking:. Fixed in fef81373696d5dc2929b40dba0343afe855c9a7f.